src/SourceBuild/content/eng/pipelines/source-build-sdk-diff-tests.yml

```diff
@@ -16,6 +9,8 @@ resources:
         include:
           - release/*.0.1xx*
           - internal/release/*.0.1xx*
+        exclude:
+          - release/9.0.1xx-preview5

```

Comment:
Isn't the assumption that this will be ready for P5?

Reply:
We're having a meeting on Monday to discuss the timeline. I figured that I would err on the side of caution with this.

Reply:
I don't think we should merge with this.  If this is needed, it should be added to the P5 branch not main.